peers2/discover_period.go

```diff
@@ -0,0 +1,92 @@
+package peers2
+
+import (
+	"sync"
+	"time"
+)
+
+const (
+	// defaultFastSync is a recommended value for aggressive peers search.
+	defaultFastSync = 3 * time.Second
+	// defaultSlowSync is a recommended value for slow (background) peers search.
+	defaultSlowSync = 30 * time.Second
+	// defaultFastSyncTimeout tells how long the fast sync should last.
+	defaultFastSyncTimeout = 30 * time.Second
```

Comment:
i thought that we won't need fastsynctimeout anymore. only the general timeout for discovery.

Reply:
I understood that you said we need both. So, you mean that it's enough to switch from `fast` to `slow` when minimal number of peers for a given topic is reached and `defaultFastSyncTimeout` is not needed at all?

Reply:
sorry for the confusion. i meant that in my opinion, for sure, need a timeout for the discovery itself, not for the switch from fast to slow. 
and we also will benefit from fast and slow timer options, but timeout for the switch from fast to slow is optional, and for simplicity can be removed.

Reply:
No problem! Thanks for clarification. I will remove it.